api/src/main/java/org/apache/iceberg/transforms/Timestamps.java

```diff
@@ -66,7 +68,7 @@ public Integer apply(Long timestampMicros) {
 
   private final ChronoUnit granularity;
   private final String name;
-  private final SerializableFunction<Long, Integer> apply;
+  private final Apply apply;
```

Comment:
we're using the same definion in https://github.com/apache/iceberg/compare/master...nastra:iceberg:fix-immutable-enum-warn?expand=1#diff-dd0f2b50502667dbbac8ee8f4eba61bffe1b9c5cc23a9445ded9fd8c725db978R68